### ISSUE TITLE
println needs a message

### ISSUE THREAD

**Issue by ph98** [hyochan/react-native-audio-recorder-player#283]
> Please use the 

### Version of react-native-audio-recorder-player
both latest on github and npm

### Platforms you faced the error (IOS or Android or both?)
android 10

### Expected behavior
work!

### Actual behavior
![image](https://user-images.githubusercontent.com/6303786/117106356-f7daaa80-ad94-11eb-8ccb-6a7bbad6aa92.png)

### Steps to reproduce the behabior
here is the code that i'm using: 
```
<TouchableOpacity
              onPressIn={() => {
                if (Platform.OS === 'android') {
                  PermissionsAndroid.requestMultiple([
                    PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE,
                    PermissionsAndroid.PERMISSIONS.READ_EXTERNAL_STORAGE,
                    PermissionsAndroid.PERMISSIONS.RECORD_AUDIO,
                  ]).then((grants) => {
                    console.log('write external stroage', grants);

                    if (
                      grants['android.permission.WRITE_EXTERNAL_STORAGE']
                          === PermissionsAndroid.RESULTS.GRANTED
                        && grants['android.permission.READ_EXTERNAL_STORAGE']
                          === PermissionsAndroid.RESULTS.GRANTED
                        && grants['android.permission.RECORD_AUDIO']
                          === PermissionsAndroid.RESULTS.GRANTED
                    ) {
                      console.log('Permissions granted');
                      audioRecorderPlayer.startRecorder().then((r) => {
                        console.log('r', r);
                      }).catch((e) => {
                        console.log('e', e);
                      });
                    } else {
                      console.log('All required permissions not granted');
                    }
                  });
                }
              }}
              onPressOut={() => {
                audioRecorderPlayer.stopRecorder().then(() => {
                  setDisabled(false);
                  setdisablePlay(false);
                }).catch((err) => {
                  console.log('err', err);
                });
              }}
            >
              <Image source={MicrophoneIcon} style={{ height: 100, width: 100 }} />
            </TouchableOpacity>
```
works fine in android 9 but not working in android 10 and showing that error

**Comment by hyochan** [comment]
@ph98 Accessing `sdcard` in Android 10 seems to break followed by [StackOverflow](https://stackoverflow.com/questions/6018633/nullpointerexception-println-needs-a-message-in-android).

Currently, please follow the [TIPS in the readme](https://github.com/hyochan/react-native-audio-recorder-player#tips).

**Comment by PepeValenzuela98** [comment]
I'm gettin the same error in the 3.0.0 release except in android 10 version
const voice = async () => {
      try {
        if (recordAudio) {
          await Voice.start('es-MX');
          await audioRecorderPlayer.startRecorder();
        } else {
          await Voice.stop();
          await audioRecorderPlayer.stopRecorder();
          if (answer) {
            dispatch(setAnswerAction({enunciate: text.join(' '), answer}));
          }
        }
      } catch (error) {
        console.log(error);
      }
    };

![image](https://user-images.githubusercontent.com/54970119/118576385-07230600-b74e-11eb-942c-20d48ea2db99.png)

**Comment by hyochan** [comment]
Fixed in `3.0.11`.
